tracim/views/core_api/workspace_controller.py

```diff
@@ -145,21 +151,24 @@ def create_generic_empty_content(
         )
         content = api.create(
             label=creation_data.label,
-            content_type=creation_data.content_type_slug,
+            content_type=creation_data.content_type,
             workspace=request.current_workspace,
         )
         api.save(content, ActionDescription.CREATION)
         content = api.get_content_in_context(content)
         return content
 
-    @hapic.with_api_doc()
+    @hapic.with_api_doc(tags=[WORKSPACES_ENDPOINTS_TAG])
```

Comment:
rename `WORKSPACES_ENDPOINTS_TAG` to `WORKSPACE_ENDPOINT_TAG` - this avoid multiple plural form which is source of typo

Reply:
Done for Users too